Dev (#84)
dl-solarity/hardhat-zkit: README.md

Reviewer: Let's tick the witness substitution checkbox.

    - `verifiersDir` - The directory where to generate the Solidity | Vyper verifiers.
    - `verifiersType` - The option (`"sol"` or `"vy"`) to indicate which language to use for verifiers generation.
- `typesDir` - The directory where to save the generated typed circuits wrappers.

> - `typesWitnessLimit` - The maximum number of witness signals for which witness signal names type will be generated.

Reviewer: ```suggestion
- `typesWitnessLimit` - The maximum number of witness substitution signals to generate types for.
```
